src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java

```diff
@@ -391,20 +393,28 @@ private boolean hasValidationAndLoadErrors() {
 
     /**
      * Has this feed expired?
-     * @return whether feed version is outside of expiration
+     * @return If the validation result last calendar date is null or has expired return true, else return false.
      */
     private boolean hasFeedVersionExpired() {
         return validationResult.lastCalendarDate == null ||
             LocalDate.now().isAfter(validationResult.lastCalendarDate);
     }
 
     /**
-     * Has this feed version produced any high severity error types when being validated?
+     * Has this feed version produced any high severity error types when being validated? Some very rural services have
+     * high priority errors for valid stops that aren't near population centers. These acceptable high priority errors
+     * are ignored.
      * @return whether high severity error types have been flagged.
      */
     private boolean hasHighSeverityErrorTypes() {
+        Set<String> acceptableHighSeverityErrorTypes
+            = new HashSet<>(Arrays.asList(
+                NewGTFSErrorType.ROUTE_UNUSED.name(),
+                NewGTFSErrorType.STOP_GEOGRAPHIC_OUTLIER.name(),
+                NewGTFSErrorType.STOP_LOW_POPULATION_DENSITY.name()));
```

Comment:
I'm not a fan of this approach... we just need to remove these error types from the high severity class in gtfs-lib. I think we discussed this early on.

Reply:
Updated and created https://github.com/conveyal/gtfs-lib/pull/310 to remove error types from high severity class in gtfs-lib.